prisma/schema.prisma

```diff
@@ -34,7 +40,7 @@ model EventType {
   eventName               String?
   customInputs            EventTypeCustomInput[]
   timeZone                String?
-  periodType              String                 @default("unlimited") // unlimited | rolling | range
+  periodType              PeriodType             @default(UNLIMITED)
```

Comment:
Add comment what this means - I don't know what `periodType` is, if it's relevant to only user or only team or both